src/main/java/net/fabricmc/example/mixin/ExampleMixin.java

```diff
@@ -1,15 +1,25 @@
 package net.fabricmc.example.mixin;
 
+import net.minecraft.client.gui.screen.Screen;
 import net.minecraft.client.gui.screen.TitleScreen;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 @Mixin(TitleScreen.class)
-public class ExampleMixin {
-	@Inject(at = @At("HEAD"), method = "init()V")
+public abstract class ExampleMixin extends Screen {
+	private MixinTitleScreen(Text title) {
+		super(title);
+	}
+	
+	@Inject(method = "init()V", at = @At("HEAD"))
 	private void init(CallbackInfo info) {
 		System.out.println("This line is printed by an example mod mixin!");
 	}
+	
+	@Inject(method = "render(IIF)V", at = @At("RETURN"))
+	private void render(int mouseX, int mouseY, float delta, CallbackInfo info) {
+		this.font.draw("Fabric Test Mod", 2, this.height - 30, -1);
```

Comment:
Bet people are going to leave it in ?

Reply:
I don think we really need this, this isnt here to be an example on how to use mixins, its an example of how to register a mixin in the json files.

This will also make it more likely to require changes when doing an update., as idealy the example mod should run on all versions. (We were very unlucky in 1.14-1.15)